### PR TITLE
Set14

### PR DESCRIPTION
**Pull request by MaximeSongIdris**
### Checks to be done before submitting your PR
- [x] `python3 -m pytest tests/` runs successfully.
- [x] `black .` runs successfully.
- [x] `make html` runs successfully (in the `docs/` directory).
- [x] Updated docstrings related to the changes (as applicable).
- [ ] Added an entry to the [CHANGELOG.rst](../CHANGELOG.rst).

### CODE REVIEW

File: deepinv/datasets/set14.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    calculate_md5_for_folder,
+    download_zipfile,
+    extract_zipfile,
+)
+
+
+class Set14HR(torch.utils.data.Dataset):
+    """Dataset for `Set14 <https://paperswithcode.com/dataset/set14>`_.
+
+    The Set14 dataset is a dataset consisting of 14 images commonly used for testing performance of Image Super-Resolution models.
```
Comment (tachella) [inline deepinv/datasets/set14.py:17]: Image Super-Resolution models -> image reconstruction algorithms

Comment (tachella) [reply deepinv/datasets/set14.py:17]: Add a line saying "Images have sizes ranging from 276×276 to 512×768 pixels."

**Review by tachella — CHANGES_REQUESTED** [review]
hey maxime, the PR looks good, I just have some minor comments on the docstrings.

File: deepinv/datasets/set14.py (hunk 1 above)
Comment (MaximeSongIdris) [reply deepinv/datasets/set14.py]: Done

**Review by tachella — APPROVED** [review]

**Pull request merged**